Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/aws/aws-sdk-go"
+  version = "1.13.11"
```

Comment:
I think we should be providing the most widely accepted version range possible for our dependencies -- especially if this is to be able to be consumed as a library

Reply:
Ok, I will make this all >= instead of =.

Reply:
Actually, this is not necessary. From the `dep` docs:

> Note: When you specify a version without an operator, dep automatically uses the ^ operator by default. dep ensure will interpret the given version as the min-boundary of a range, for example:
>
>1.2.3 becomes the range >=1.2.3, <2.0.0
>0.2.3 becomes the range >=0.2.3, <0.3.0
>0.0.3 becomes the range >=0.0.3, <0.1.0

Reply:
ah, good to know. Then we should just try to figure out what the oldest version of the library is we can support. It's really frustrating as a developer to end up in go vendor hell because a transitive dependency is off by a patch version :)